src/main/java/bio/terra/service/filedata/google/firestore/FireStoreDirectoryDao.java

```diff
@@ -345,6 +345,13 @@ List<FireStoreDirectoryEntry> query(Query query) throws InterruptedException {
     return fireStoreUtils.query(query, FireStoreDirectoryEntry.class);
   }
 
+  List<FireStoreDirectoryEntry> enumerateFileRefEntries(
+      Firestore firestore, String collectionId, int offset, int limit) throws InterruptedException {
+    Query fileColl = firestore.collection(collectionId).whereEqualTo("isFileRef", true);
+    return fireStoreUtils.query(fileColl, FireStoreDirectoryEntry.class, offset, limit).stream()
+        .toList();
```

Comment:
Why does this use `.stream().toList()`? `query()` returns a `List<FireStoreDirectoryEntry>`.
```suggestion
    return fireStoreUtils.query(fileColl, FireStoreDirectoryEntry.class, offset, limit);
```